packages/libglib/libglib.spec

```diff
@@ -9,12 +9,10 @@ BuildRequires: meson
 BuildRequires: %{_cross_os}glibc-devel
 BuildRequires: %{_cross_os}libffi-devel
 BuildRequires: %{_cross_os}libmount-devel
-BuildRequires: %{_cross_os}libpcre-devel
 BuildRequires: %{_cross_os}libselinux-devel
 BuildRequires: %{_cross_os}libz-devel
 Requires: %{_cross_os}libffi
 Requires: %{_cross_os}libmount
-Requires: %{_cross_os}libpcre
```

Comment:
We should also remove this from the `Cargo.toml` and regenerate `Cargo.lock` under variants.